packages/web-runtime/src/components/Topbar/Notifications.vue

```diff
@@ -70,8 +58,12 @@
 import { mapGetters, mapActions } from 'vuex'
 import { eventBus } from 'web-pkg/src/services/eventBus'
 import { ShareStatus } from 'web-client/src/helpers/share'
+import NotificationBell from './NotificationBell.vue'
 
 export default {
+  components: {
+    NotificationBell
+  },
   computed: {
     ...mapGetters(['activeNotifications', 'configuration']),
```

Comment:
`configuration` getter is unused, please remove (not touched by you, but still a good chance to clean up)